src/app/components/harvest/screens/uploading/batch-uploading.component.html

```diff
@@ -51,6 +76,40 @@ <h3>Uploading Files</h3>
       </baw-wip>
     </ng-template>
   </li>
+  <li [ngbNavItem]="4">
+    <a ngbNavLink>R Clone</a>
+    <ng-template ngbNavContent>
+      <ol>
+        <li>
+           Rclone is a <b>command line</b> tool for transferring files, which you can download 
+           and install <a target="_blank" href="https://rclone.org/install/">here</a>. 
+        </li>
+        <li>
+          Paste the following command into your terminal, making sure to replace the [path to your files]    
+          <ul>
+            <li>
+              <code>
+                rclone copy -v 
+                --sftp-user {{ harvest.uploadArguments.user }} 
+                --sftp-pass $(rclone obscure {{ harvest.uploadArguments.password }}) 
+                --sftp-host {{ harvest.uploadArguments.host }} 
+                --sftp-port {{ harvest.uploadArguments.port }} 
+                "[path to your files]"
+                :sftp:/
+              </code>   
+            </li>
+
+            <li>
+              Note that rclone will copy the <em>contents</em> of the [path to your files] folder to the destination, not the source folder itself.
+              So if you have a folder for each site, specify their parent folder as the [path to your files] value. 
```

Comment:
wrap `[path to your files]` in an inline code style

Reply:
done